Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1271,14 +1271,6 @@ theorem eLpNorm'_le_mul_eLpNorm'_of_ae_le_mul {f : α → ε} {c : ℝ≥0∞} {
     simp [ENNReal.mul_rpow_eq_ite, enorm_eq_zero, hp']
   simpa [ENNReal.coe_rpow_of_nonneg _ hp.le, aux, ENNReal.rpow_le_rpow_iff hp]
 
-theorem eLpNormEssSup_le_nnreal_smul_eLpNormEssSup_of_ae_le_mul {f : α → F} {g : α → G} {c : ℝ≥0}
```

Comment:
Oops, these hunks also snuck in. They are just re-ordering lemmas
- the first one is moved outside an enormed block (as the lemma uses normed spaces)
- secondly, move the results so the enorm ones come first (this will potentially golfing the normed proofs, and also signals that enorm ones might be a tad better)